api/v1/helpers/verbs/utils.js

```diff
@@ -238,20 +240,46 @@ function handleAssociations(reqObj, inst, props, method) {
 } // handleAssociations
 
 /**
- * Generates sequelize options object with all the appropriate attributes
+ * Generates Sequelize options object with all the appropriate attributes
  * (fields) and includes, and taking virtual fields into account as well.
- * Always include the "id" field even if it was not explicitly requested.
+ * Include always:
+ *  The "id" field even if it was not explicitly requested.
+ *  All table foreign keys to avoid multiple association nested queries
+ *  failures.
  *
  * @param {Object} params - The request parameters
- * @returns {Object} - Sequelize options
+ * @param {Object} model - The helpers/nouns DB model
+ * @returns {Object} - option containing either attributes to be used
+ *  in Sequelize and foreign keys to be removed when send back to the client.
  */
-function buildFieldList(params) {
+function buildFieldList(params, model) {
   const opts = {};
   if (params.fields && params.fields.value) {
     opts.attributes = params.fields.value;
     if (!opts.attributes.includes('id')) {
       opts.attributes.push('id');
     }
+
+    /*
+     * Includes all model's FK for an eventual outer join.
+     */
+    if (model) {
+      opts.fieldsToExclude = [];
+
```

Comment:
why empty line here? i don't feel like it adds to readability (but it's fine if you do)